.github/workflows/reusable_multi_numa.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
```

Comment:
as I mentioned before - pls update commits, to only have a single one, with proper description. Message `Update reusable_multi_numa.yml` doesn't say much 😉